src/assets/SASSAsset.js

```diff
@@ -24,9 +24,10 @@ class SASSAsset extends Asset {
     let opts =
       (await this.getConfig(['.sassrc', '.sassrc.js'], {packageKey: 'sass'})) ||
       {};
-    opts.includePaths = (opts.includePaths || []).concat(
-      path.dirname(this.name)
-    );
+    opts.includePaths = (opts.includePaths
+      ? opts.includePaths.map(includePath => path.resolve(includePath))
```

Comment:
this resolves paths relative to the current working directory. is this right, or do we want to resolve relative to the sass file being compiled?

Reply:
Yes, only reative to the current working directory will not change original behavior. It seems that the sass compiler has the same logic for reative path in `includePaths`.